.eslintrc.yml

```diff
@@ -0,0 +1,24 @@
+extends:
+  - '@inveniosoftware/invenio'
+  - '@inveniosoftware/invenio/prettier'
+
+parser: '@babel/eslint-parser'
+
+rules:
+  # allow default prop to be set even if prop required
+  react/default-props-match-prop-types: 0
```

Comment:
The idea is to not add custom rules to each repo but to use the ones provided by the eslint repo to make sure that all the projects follow the same rules. That said if a rule is meant to be added should be in the [eslint repo](https://github.com/inveniosoftware/eslint-config-invenio). We already defined [here](https://github.com/inveniosoftware/eslint-config-invenio/blob/master/configs/main.yaml#L22) this rule to fail on purpose, why would we want to allow default props for required props?

Reply:
That is a very good point, i did not realize it was set on purpose . I set it here because it seems to be a common pattern for some props and i was not sure about changing the actual code aside from formatting and some minor code refactoring (destructuring prop, adding prop types).

Reply:
on another subject: do we have any specification on the line lenght ? It would be good to align with black(backend formatter)

Reply:
We use the default value which is 80 if I am not mistaken , but we can update it.

Reply:
after the discussion, during the stand-up today, I think in the end a prop which has a default is not technically required, since it is not provided when the component is used, so it makes sense to have either default or required, not both

Reply:
No, not added. We quickly discussed during the stand-up and I personally agree with what @kpsherva explained in the previous comment, the idea is to keep the rule as it is, however this is open to discussion, might be faster IRL.

Reply:
indeed as @jrcastro2 said